Podfile

```diff
@@ -1,11 +1,7 @@
 # Uncomment the next line to define a global platform for your project
-platform :ios, '13.0'
+platform :ios, '14.0'

```

Comment:
Does this break support for 13 / was there a reason for bumping?

Reply:
https://github.com/DestinyItemManager/DIM/pull/8933  partially related?

Reply:
We increased the minimum iOS version for DIM to 14.0 a while back.